instagram/src/components/Login/Login.js

```diff
@@ -0,0 +1,40 @@
+import React, {Component} from 'react';
+
+
+class Login extends Component{
+    constructor(props) {
+        super(props); 
+        this.state = {
+            user: '',
+            password: ''
+        }
+    }
+
+    inputHandle = (event) =>{
+        this.setState({
+            [event.target.name]: event.target.value
+        });
+    };
+
+    submitHandle = (event) =>{
+        const user = this.state.user;
+        localStorage.setItem('user', user);
+        window.location.reload();
+    }
+
+    render(){
+        return(
+            <div>
+                <form>
+                    <input type='text' placeholder='Username' name='user' value={this.state.user} onChange={this.inputHandle} />
+                    <input type='text' placeholder='Password' name='password' value={this.state.password} onChange={this.inputHandle} />
```

Comment:
Set the input type of the password to 'password' instead of 'text' so the password is not shown when typing out